plotter/step_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright ©2018 The gonum Authors. All rights reserved.
```

Comment:
s/gonum/Gonum/

Reply:
Shouldn't it be changed in some bulk way for the entire repository? Currently non-capitalized version is used in every other file.

Reply:
OK